ckanext/cnra_schema/fanstatic/css/composite_display_snippet.css

```diff
@@ -0,0 +1,4 @@
+.cnra_composite_display_snippet_dl {
+    margin-bottom: -10px;
+    margin-top: 10px;
+}
```

Comment:
Should there be a 10px margin at the top?
Also I left a comment on the composite PR about the class naming.

Reply:
If 10px margin is not there then there is a bigger gap from the beginning text to the top of the cell